modules/core/src/errors.ts

```diff
@@ -130,3 +130,10 @@ export class EthereumLibraryUnavailableError extends BitGoJsError {
     Object.setPrototypeOf(this, EthereumLibraryUnavailableError.prototype);
   }
 }
+
+export class StellarFederationUserNotFoundError extends BitGoJsError {
```

Comment:
Since you're modifying this file anyway, could you fix the incorrect prototype set on line 123? (should be setting to `AddressGenerationError.prototype` instead of `KeyRecoveryServiceError.prototype`).